sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1334,6 +1334,15 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val typeCoercionMode =
+    buildConf("spark.sql.typeCoercion.mode")
+      .doc("Since Spark 2.4, the 'hive' mode is introduced for Hive compatiblity. " +
+        "Spark SQL has its native type cocersion mode, which is enabled by default.")
```

Comment:
I like that you added documentation.  Did you mean "coercion" rather than "cocersion"?